AzureCalling/app/src/main/java/com/azure/samples/communication/calling/contracts/SampleErrorMessages.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.contracts;
+
+public class SampleErrorMessages {
+    public static final String GROUP_ID_REQUIRED = "Group call ID is required";
+    public static final String GROUP_ID_INVALID = "Group call ID is not valid";
+    public static final String DISPLAY_NAME_REQUIRED = "Display Name is required";
+    public static final String TEAMS_LINK_REQUIRED = "Teams Link is required";
+    public static final String TEAMS_LINK_INVALID = "Teams Link is not valid";
+    public static final String CALL_COMPOSITE_JOIN_CALL_FAILED = "Could not join call";
+    public static final String CALL_COMPOSITE_END_CALL_FAILED = "Could not end call";
+    public static final String CALL_COMPOSITE_TOKEN_EXPIRED = "ACS Token has expired";
+}
```

Comment:
Please create task to refactor to use XML.

Reply:
Noted.